.github/labeler.yml

```diff
@@ -0,0 +1,34 @@
+# https://github.com/actions/labeler
+
+# Add 'repo' label to any root file changes
+"Area: Repo":
+  - "*"
+
+"Area: Components":
+  - packages/paste-core/components/**/*
+
+"Area: Doc Site":
+  - packages/paste-website/**/*
+
+"Area: Theme":
+  - packages/paste-theme/**/*
+
+"Area: Tokens":
+  - packages/paste-design-tokens/**/*
+
+"Area: Storybook":
+  - .storybook/**/*
+
+"Area: Infrastructure":
+  - .eslint/**/*
+  - .github/**/*
+  - .jest/**/*
+  - .vscode/**/*
+  - tools/**/*
+
+"Type: Documentation":
+  - packages/paste-website/src/pages/**/*
+
+"Type: Tests":
```

Comment:
Does this mean that there can be both a Type and Area label for a PR?

Reply:
Yup, you can have as many labels as you like